qa/rpc-tests/import-rescan.py

```diff
@@ -127,10 +127,19 @@ def setup_network(self):
         self.nodes = start_nodes(self.num_nodes, self.options.tmpdir, extra_args)
         for i in range(1, self.num_nodes):
             connect_nodes(self.nodes[i], 0)
+        self.sync_all()
+
+    def sync_all(self):
```

Comment:
if we're not synchronizing "all", perhaps this should be called something different than `sync_all`?

Reply:
Good point. Renamed.